tests/st/calicoctl/test_crud.py

```diff
@@ -627,7 +635,8 @@ def test_create_from_file(self, res, data1, data2):
                                      'ports': [1, 2, 3, 4],
                                      'tag': 'web'}}],
                     'order': 6543215.321,
-                    'selector': ''}},
+                    'selector': '',
+                    'types': ['ingress', 'egress']}},
```

Comment:
All of these tests are both `ingress` and `egress` - can we add a few new tests to validate ingress-only, and egress-only policies?

Also, a policiy that has no ingress or egress rules defined?

Reply:
The second test `pop`s the egress rule from policy1 and confirms that no `types` field is appended. I could make this more clear by creating another policy_dictionary, if necessary.

I'll add another egress-only test as well as a no-ingress and no-egress policy.

Note that there are already dozens of these tests in other sections that are effectively testing this behavior now as well.